### DISCUSSION TITLE
How make it work for a Digital Display Meter with screen content changes?

### DISCUSSION THREAD

**Discussion by jnanadarshan** [jomjol/AI-on-the-edge-device#2183] [Q&A] [answered]
I have a Meter with a Digital Display where the Display changes every couple of seconds to show Meter Reading, Time, Current Load in KW, etc. How can I use this for the Digital Meter? These kind of meters are failry common in India.

![20230314_170512](https://user-images.githubusercontent.com/16976738/224989730-c50eb944-7783-4972-ac3b-c88b90b218ac.jpg)
![20230314_170545](https://user-images.githubusercontent.com/16976738/224989737-383a1ecb-2ff4-490a-ae13-ff16267d5bc6.jpg)
![20230314_170525](https://user-images.githubusercontent.com/16976738/224989740-851b4902-8f23-4dab-b3d9-c133f6f66a61.jpg)

**Comment by caco3** [comment, marked as answer]
Detecting through iterating is currently not supported. See https://github.com/jomjol/AI-on-the-edge-device/issues/1936

The only way you currently have is to take the pictures in a higher rate and use the rate limit and consistency checks to make sure only one of the 3 modes provides valid data.
If you want to detect multiple modes, you could use multiple numbers and place the ROIS on top of each other
